server/fishtest/userdb.py

```diff
@@ -114,6 +120,7 @@ def create_user(self, username, password, email):
 
             return True
         except:
+            request.session.flash("Error! Invalid username or password", "error")
```

Comment:
This will not work no? `create_user()` does not know about `request`. Besides `userdb` is for handling the db, not for user interaction.

Reply:
> This will not work no? `create_user()` does not know about `request`. Besides `userdb` is for handling the db, not for user interaction.

Please check now

Reply:
This seems ok!